prx_simulation/control_msg[] plan
string system_name
---
